src/commands/autoview.rs

```diff
@@ -170,3 +201,25 @@ pub fn autoview(
         }
     }))
 }
+
+fn create_default_command_args(context: &RunnableContextWithoutInput) -> RawCommandArgs {
+    let span = context.name.span;
+    RawCommandArgs {
+        host: context.host.clone(),
+        ctrl_c: context.ctrl_c.clone(),
+        shell_manager: context.shell_manager.clone(),
+        call_info: UnevaluatedCallInfo {
+            args: hir::Call {
+                head: Box::new(SpannedExpression::new(
+                    Expression::Literal(Literal::String(span)),
+                    span,
+                )),
+                positional: None,
+                named: None,
+                span,
+            },
+            source: context.source.clone(),
+            name_tag: context.name.clone(),
+        },
+    }
```

Comment:
😅 reading this part makes me realize there are a lot of concepts I still need to learn.

Reply:
It's, umm, not the prettiest thing admittedly :sweat_smile: